api/routes.go

```diff
@@ -73,8 +73,8 @@ func (s *server) routes() {
 	api.HandleFunc("/{account}/instances/{id}/attribute", s.ProxyRequestHandler).Methods(http.MethodPut)
 	api.HandleFunc("/{account}/sgs/{id}", s.SecurityGroupUpdateHandler).Methods(http.MethodPut)
 	api.HandleFunc("/{account}/sgs/{id}/tags", s.ProxyRequestHandler).Methods(http.MethodPut)
-	api.HandleFunc("/{account}/volumes/{id}", s.ProxyRequestHandler).Methods(http.MethodPut)
-	api.HandleFunc("/{account}/volumes/{id}/tags", s.ProxyRequestHandler).Methods(http.MethodPut)
+	api.HandleFunc("/{account}/volumes/{id}", s.VolumeUpdateHandler).Methods(http.MethodPut)
+	api.HandleFunc("/{account}/volumes/{id}/tags", s.VolumesTagsHandler).Methods(http.MethodPut)
```

Comment:
Why don't we do the same thing we did for instances and use `VolumeUpdateHandler` for both of these?

Reply:
Done